api/tests/unit/application/session/index_test.js

```diff
@@ -33,6 +32,7 @@ describe('Unit | Application | Sessions | Routes', () => {
     sinon.stub(sessionController, 'flagResultsAsSentToPrescriber').returns('ok');
     sinon.stub(sessionController, 'assignCertificationOfficer').returns('ok');
 
+    server = this.server = Hapi.server();
```

Comment:
c'est hors scope, mais le this.server, il sert à quoi?

Reply:
une tentative avortée oubliée ?

Reply:
je l'ai supprimé à ça ne change rien